index.js

```diff
@@ -1 +1,37 @@
-// Add your code here
+const postData = {
+  name: "Yuriy",
+  email: "yuriy.ivanenko@powerhrg.com",
+};
+
+const handlePostRespone = (data) => {
+  const div = document.querySelector('#respone_data')
+  const p = document.createElement('p')
+  p.textContent = `Respone ID: ${data.id}`
+  div.appendChild(p)
+} 
+
+const handlePostError = (error) => {
+  const div = document.querySelector('#respone_data')
+  const p = document.createElement('p')
+  p.textContent = `Error: ${error}`
+  div.appendChild(p)
+}
+
+const submitData = (user_name, user_email) => {
+  const reqBody = {
+    name: user_name,
+    email: user_email
+  }
+  const headers = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json'
+  }
+return fetch('http://localhost:3000/users',{
```

Comment:
I don't think you need to return the fetch, although I doesn't hurt anything. 
```suggestion
  fetch('http://localhost:3000/users',{
```